Makefile

```diff
@@ -1,31 +1,47 @@
 .RECIPEPREFIX +=
 
 # Run Postgres
+.PHONY: db
 db:
   docker run --name codesearch-db \
```

Comment:
It would be good add volume for folder `var/lib/postgresql/data`
Volume create in another step